### ISSUE TITLE
In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are not visible in windows.

### ISSUE THREAD

**Issue by msftedad** [twbs/bootstrap#42032]
### Prerequisites

- [x] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7653)
Browser: Edge (Version 144.0.3719.82)
URL: https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ

**User Impact:**
In high contrast themes (Aquatic, Desert, Dusk, Night Sky), users with low vision or those relying on keyboard navigation cannot identify the selected date or focus indicator on calendar controls because they are not visible. This makes scheduling and date selection difficult, reducing accessibility and usability.

**Pre-Requisites:**
Setting > Accessibility > Contrast Themes > (Aquatic/Desert/Dusk/Night Sky)

**Repro Steps:**

1. Open the URL: https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ in edge browser in windows.
2. Press tab key to navigate to the 'Select Date' control.
3. Calendar will open. 
4. Press tab key to navigate to the calendar dates. 
5. Verify whether in high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are visible in windows or not.

**Actual Result:**
In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are not visible in windows.

**Refer Attachment:**

1. In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are not visible in windows.png
2. In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls are not visible in windows.mp4

https://github.com/user-attachments/assets/6eb55d55-5bb3-464a-9449-994eb9218826
<img width="2945" height="955" alt="Image" src="https://github.com/user-attachments/assets/430e1379-65bd-4c76-9fa0-fd80712c0ab1" />

**Expected Result:**
In high contrast themes (Aquatic, Desert, Dusk, Night Sky), the selected date and focus indicator on calendar controls should be visible in windows.

### Reduced test cases

https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What version of Bootstrap are you using?

1.10.0

**Comment by julien-deramond** [comment]
Hello @msftedad 
Closing as bootstrap-datepicker is not handled by the Bootstrap core team. You can create your issue in their GitHub repository. Not entirely sure, to double-check, but it's possibly this one: https://github.com/uxsolutions/bootstrap-datepicker
